README.md

```diff
@@ -75,6 +75,8 @@ make install NAMESPACE=test-namespace
 
 ### Submitting Spark Application
 
+To get started with your app monitoring, please, see also [MONITORING.md](MONITORING.md) 
```

Comment:
it makes sense to place this reference at the end of the section

Reply:
done